lib/core.yaml

```diff
@@ -8,13 +8,15 @@ linter:
     - camel_case_extensions
     - camel_case_types
     - curly_braces_in_flow_control_structures
+    - depend_on_referenced_packages
```

Comment:
Auto fix for this one is still under discussion, see https://github.com/dart-lang/lints/issues/42.